robot:
  rate: 50
  debug: true
  arm1:
    base:
      actuator:
        controller: 'motor'
        enabled: true
        topic: 'pwm'
        lpwm: 0
        rpwm: 1
        minPwm: 64
        maxPwm: 255
        minVelocity: 0.0
        maxVelocity: 1.0
      encoder:
        controller: 'encoder'
        enabled: true
        topic: 'adc'
        channel: 0
        minReading: 810
        maxReading: 258
        minPos: -1.4929
        maxPos: 1.4929
        threshold: 8
        average: 2
    upperarm_actuator:
      actuator:
        controller: 'motor'
        enabled: true
        topic: 'pwm'
        lpwm: 2
        rpwm: 3
        minPwm: 32
        maxPwm: 255
        minVelocity: 0.0
        maxVelocity: 1.0
      encoder:
        controller: 'encoder'
        enabled: true
        topic: 'adc'
        channel: 1
        minReading: 857
        maxReading: 902
        minPos: -0.054
        maxPos: -0.0005
        threshold: 8
        average: 2
    forearm_actuator:
      actuator:
        controller: 'motor'
        enabled: true
        topic: 'pwm'
        lpwm: 4
        rpwm: 5
        minPwm: 200
        maxPwm: 255
        minVelocity: 0.0
        maxVelocity: 1.0
      encoder:
        controller: 'encoder'
        enabled: true
        topic: 'adc'
        channel: 6
        minReading: 976
        maxReading: 848
        minPos: -0.0508
        maxPos: 0
        threshold: 8
        average: 8
    wrist:
      actuator:
        controller: 'solenoid'
        enabled: true
        topic: 'pwm'
        channel: 6
        triggerSeconds: 0.023
